### PR TITLE
Create one single page listing all AMPHTML ads for easier manual testing

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: examples/amphtml-ads/many-ads.host.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    }
+    a {
+      display: block;
+    }
+  </style>
+</head>
+<body>
+<h3>Image ad w/ Ad Choices</h3>
+<iframe
+    src="//ads.localhost:8000/examples/amphtml-ads/adchoices-1.a4a.html"
```
Comment (powerivq) [inline examples/amphtml-ads/many-ads.host.html]: Just question, is it intended to create a cross-domain iframe?

**Comment by powerivq** [comment]
`gulp --extensions_from=examples/xxx.html` does not seem to work with inabox. (ref https://github.com/ampproject/amphtml/blob/master/build-system/tasks/extension-helpers.js#L188)

@lannka Maybe fix it in the fix-it-week?

File: examples/amphtml-ads/many-ads.host.html (hunk 1 above)
Comment (lannka) [reply examples/amphtml-ads/many-ads.host.html]: yes. this is cross-domain inabox rendering

**Comment by lannka** [comment]
> gulp --extensions_from=examples/xxx.html does not seem to work with inabox doc

You can sync to head (#23615) and do: `gulp --with_inabox --extensions=inabox`

**Pull request merged**